src/non_nil.rs

```diff
@@ -33,9 +33,15 @@ use crate::{
 /// may change. It is currently only guaranteed that `NonNilUuid` and `Option<NonNilUuid>`
 /// are the same size as `Uuid`.
 #[repr(transparent)]
-#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
+#[derive(Copy, Clone, PartialEq, Eq, Hash)]
 pub struct NonNilUuid(NonZeroU128);
 
+impl fmt::Debug for NonNilUuid {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "{}", Uuid::from(*self))
```

Comment:
I think it would be better to write this as:

```
fmt::Display::fmt(&Uuid::from(*self), f)
```

This uses less of the formatting machinery and preserves any flags on `f`.

Reply:
I've changed it to directly calling `Debug::fmt`, and also updated the `Display` implementation to do the same. My change was incorrect as well (should have been using `"{:?}"` instead of `"{}"`), but that should be fixed with the direct calls as well.